homeassistant/components/switchbee/button.py

```diff
@@ -26,24 +26,13 @@ async def async_setup_entry(
     )
 
 
-class SwitchBeeButton(CoordinatorEntity[SwitchBeeCoordinator], ButtonEntity):
+class SwitchBeeButton(SwitchBeeEntity, ButtonEntity):
```

Comment:
Why should the button not be part of the device?

Reply:
A button does not belong to a specific device, it is a virtual scenario exposed from switchbee hub.

Reply:
The hub should have a service type device info, and the standard devices should set "via device" information in their info.

Reply:
Ps. That can be done separate. Doesnt block changes here.

Reply:
Lets keep that for followup